src/stores/__tests__/SearchStore.spec.ts

```diff
@@ -33,18 +31,40 @@ describe('settingsStore', () => {
   });
 
   it('initializes', () => {
-    expect(searchStore.searchFilters).toEqual({});
+    expect(searchStore.savedSearches).toEqual([]);
   });
```

Comment:
[For the future] We could add a test for the specific case of "translating" the old store to the new one